### ISSUE TITLE
Rokitansky-Aschoff sinuses of the gallbladder

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10044]
**Preferred term label:**
Rokitansky-Aschoff sinuses of the gallbladder

**Synonyms**
Epithelial invaginations forming cystic pockets

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0267892

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adenomyomatosis
Mirizzi syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6000450
- label: Rokitansky-Aschoff gallbladder sinuses
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:31984708
- def: Localized or diffuse epithelial proliferation with hypertrophy of the muscularis layer and invagination of the mucosa through the muscularis, forming intramural diverticula (i.e., small, bulging pouches in the wall of the gallbladder).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10044
- synonyms: Adenomyomatosis of the gallbladder(NONE)|Epithelial invaginations of the gallbladder forming cystic pockets(NONE)
